components/form/Form.tsx

```diff
@@ -48,6 +48,7 @@ export type WrappedFormUtils = {
                           callback?: (erros: any, values: any) => void): void;
   /** 获取某个输入控件的 Error */
   getFieldError(name: string): Object[];
+  getFieldsError(names?: Array<string>): Object[];
```

Comment:
The returned type should be an object, see: https://github.com/react-component/form/blob/master/src/createBaseForm.js#L253-L260